pyproject.toml

```diff
@@ -15,7 +15,7 @@ description = "Ophyd devices and other utils that could be used across DLS beaml
 dependencies = [
     "click",
     "ophyd",
-    "ophyd-async>=0.6,<0.7",
```

Comment:
Given the issues we've had with updating ophyd-async breaking everything, I really do think we should pin <0.x+1 as a point of habit until 1.0 and aim to upgrade early but without disruption.